docs/embed/use-case/wasm-smart-devices.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 3
```

Comment:
This is not the use case for embedding.

Reply:
You didn't solve the problem.

Reply:
Please remove this chapter.

Reply:
@alabulei1 I have put it under overview/start itself. Had missed pushing one change, Just did it below